src/almanack/metrics/data.py

```diff
@@ -554,6 +554,90 @@ def process_repo_for_analysis(
     finally:
         shutil.rmtree(temp_dir)
 
+import pandas as pd
+
+def _table_to_wide(table_rows: list[dict]) -> Dict[str, Any]:
+    """
+    Transpose Almanack table (name->result), compute checks summary, flatten nested.
+    File-level entropy is completely ignored.
+    """
+    if not table_rows:
+        return {
+            "checks_total": 0,
+            "checks_passed": 0, 
+            "checks_pct": None
+        }
+
+    df = pd.DataFrame(table_rows)
+
+    # Dynamic sustainability checks: bool + positive correlation
+    mask = (df["result-type"] == "bool") & (df["sustainability_correlation"] == 1)
+    checks_total  = int(mask.sum())
+    checks_passed = int((df.loc[mask, "result"] == True).sum())  # noqa: E712
+    checks_pct    = (100.0 * checks_passed / checks_total) if checks_total else None
+
+    # name->result (wide format)
+    wide: Dict[str, Any] = (
+        df[["name", "result"]]
+        .set_index("name")
+        .T
+        .reset_index(drop=True)
+        .iloc[0]
+        .to_dict()
+    )
+
+    # Remove file-level entropy completely - we don't want it
+    if "repo-file-info-entropy" in wide:
+        del wide["repo-file-info-entropy"]
+
+    # Flatten repo-almanack-score if present (avoid nested dict in parquet)
```

Comment:
Just in case: if you ran into challenges with nested values in Parquet you might consider the use of [`awkward`](https://awkward-array.org/doc/main/reference/generated/ak.to_parquet.html) (which includes a custom `parquet` format which allows for jagged/"awkward" array storage) or [`lance`](https://lancedb.github.io/lancedb/guides/tables/) (which is kind of like a `parquet 2.0` with much stronger nested value support) which could help nest these successfully.